sky/setup_files/setup.py

```diff
@@ -68,14 +69,27 @@
 
 extras_require['all'] = sum(extras_require.values(), [])
 
+long_description = ''
+readme_filepath = 'README.md'
+# When sky/backends/wheel_utils.py builds wheels, it will not contain the README.
+# Skip the description for that case.
+if os.path.exists(readme_filepath):
+    long_description = io.open(readme_filepath, 'r', encoding='utf-8').read()
```

Comment:
Does this work since this is under `sky/setup_files/` (I see there's a symlink at root)?

Reply:
Yes, it works for building the PyPI package, installing it from the PyPI and launching a new instance with the sky installed with `pip install skypilot[aws]`. Our `wheel_utils` is the only one that uses the `sky/setup_files`. Probably, after we have the PyPI setup, we can let the remote instance install `skypilot` from PyPI, instead of using wheels (tho it may make it harder for developing).